vllm/worker/embedding_model_runner.py

```diff
@@ -47,12 +46,14 @@ def __init__(
     @torch.inference_mode()
     def execute_model(
         self,
-        seq_group_metadata_list: Optional[List[SequenceGroupMetadata]],
+        broadcast_inputs: Dict[str, Any],
+        aux: Optional[List[Any]],
```

Comment:
Why do we add an `aux` argument? I feel like this makes arguments more confusing.

Reply:
how about the current design:

```
    def execute_model(
        self,
        modelrunner_input: ModelRunnerInput,
        kv_caches: List[torch.Tensor],
    ) -> Optional[SamplerOutput]:
```